cmd/applyconfig/applyconfig.go

```diff
@@ -162,10 +172,11 @@ type configApplier struct {
 	path       string
 	user       string
 	dry        dryRunMethod
+	apply      applyMethod
 	censor     *secrets.DynamicCensor
 }
 
-func makeOcApply(kubeConfig, context, path, user string, dry dryRunMethod) *exec.Cmd {
+func makeOcApply(kubeConfig, context, path, user string, dry dryRunMethod, apply applyMethod) *exec.Cmd {
```

Comment:
Why do we need the extra flag?

Reply:
In case we switch to server-side apply globally.